config/routes.rb

```diff
@@ -4,6 +4,13 @@
   get '/users', to: 'users#index'
   get '/users/:id', to: 'users#show', as: :user
 
-  get '/users/:user_id/posts', to: 'posts#index', as: :user_posts
-  get '/users/:user_id/posts/:id', to: 'posts#show', as: :user_post
+  resources :users do
+    resources :posts do
+      resources :comments, only: [:create]
+    end
+  end
+
+  post '/users/:user_id/posts', to: 'posts#create', as: :create_post
+  post '/users/:user_id/posts/:post_id/comments', to: 'posts#create_comment', as: :create_comment
+  post '/users/:user_id/posts/:id/like', to: 'posts#like', as: :like_user_post
```

Comment:
- [ ] Great job implementing the creation of likes and making it possible for a user to add a like to posts :+1: However, your implementation currently increments the `likes_counter` by 2 instead of by 1. So kindly fix this :wink: 

[Screencast from 10-14-2023 10:20:21 AM.webm](https://github.com/najibullahjafari/ruby_social_media/assets/102699958/7fe37ad4-2c41-4530-b226-d10af71bb414)